packages/scrum-client/src/hooks/ClientSocketEvents.js

```diff
@@ -41,6 +41,7 @@ const ClientSocketEvents = (ioUri = API_CONSTANTS.API_BASE_URL) => {
   const onStoryVotesUpdate = cb => {
     console.log('onStoryVotesUpdate');
     socket.on(EVENT.STORY_VOTES_UPDATE, data => {
+      console.log('onStoryVotesUpdate-data', data);
```

Comment:
Is it intended to keep this console? Or was it just for testing? I am fine with it just double checking

Reply:
to leave it until we don have more issues, then we can remove , but for now it is intended